components/products/ProductDetail.tsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
-import { Product } from "../../entities/Product";
+import { ProductItemTest } from "../../entities/Product";
 import Image from "next/image";
 
 interface ProductDetailProps {
-  product: Product;
+  product: ProductItemTest;
```

Comment:
I'm assuming this is a temporary change?

Reply:
Add a new ProductItemTest interface to resolve compatibility issues with ProductDetail.tsx but I think we need to consider updating the Product interface